docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -1,15 +1,26 @@
 # Maintenance
 ## Private Packagist Self-Hosted (Kubernetes)
 
+## kURL installations
+
 ### Updates
 
+This section describes how to update a Private Packagist Self-Hosted in a Kubernetes cluster installed with [kURL](kubernetes-embedded).
+This section is not relevant if you installed the application with [Helm](kubernetes-helm).
+
 By default Replicated checks for updates every 4 hours and you can install updates from the dashboard in the Replicated
 Management Console on port 8800 once they are available.
 You can configure this behavior under “Configure automatic updates” on the dashboard.
 
 The [Private Packagist Self-Hosted Changelog](https://packagist.com/docs/self-hosted/changelog) details all new features,
 behavior changes and major bugfixes each release introduces.
 
+You can safely bump multiple versions at once, it's not required to install individual versions consecutively. If we ever 
+release a version that needs to be installed specifically, we will make sure to communicate this first.
+
+We recommend backing up your database before each update. We cannot guarantee that downgrading to a previous release will always work
+due to database migrations. In case of upgrade failure, the most reliable way to rollback to the previous version is via backup.
```

Comment:
Note: this is the same information as added to Helm updates too, since these are relevant in both places.

Reply:
Sounds good! This might need further modifications down the road as backups in the Kots UI are called snapshots but I think this is ok for now

Reply:
@glaubinix Thanks, I've simply changed the word "backup" to "snapshot" in the KOTS paragraph for now. If more info is required we can add that later.

Reply:
Great! Thanks!